docs/git/cli/homework.md

```diff
@@ -0,0 +1,58 @@
+---
+id: homework
+title: Homework
+sidebar_label: Homework
+---
+
+## 1) Read the Lesson Page Content (60 minutes)
+
+Before starting this coursework, read through the lesson and complete the exercises.
```

Comment:
```suggestion
Before starting this coursework, read through the lesson and read the cheat-sheet.
```

There aren't any exercises in the lesson, but we should make sure they read the cheat-sheet, rather than just acknowledging it's there :)